README.md

```diff
@@ -6,7 +6,22 @@
 
 ---
 
-The IntelliJ Ballerina plugin provides the Ballerina development capabilities in IntelliJ IDEA. Click the links below for instructions on how to download, install, and use the features of the IntelliJ plugin.
+## Overview
+The Ballerina IntelliJ plugin provides the [Ballerina programming language](https://ballerina.io/) development capabilities within IntelliJ IDEA. While it offers a subset of features compared to the [Ballerina VSCode extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina), it is designed to meet the specific needs of those working within the IntelliJ ecosystem.
```

Comment:
Should we mention about the compatibility with BAllerian swan lake versions?

Reply:
+1